src/openzaak/conf/includes/base.py

```diff
@@ -568,3 +566,7 @@
     "CMIS_MAPPER_FILE", default=os.path.join(BASE_DIR, "config", "cmis_mapper.json")
 )
 CMIS_URL_MAPPING_ENABLED = config("CMIS_URL_MAPPING_ENABLED", default=False)
+
+# Notificaties API settings
+SELF_REPO = "VNG-Realisatie/notificaties-api"
+SELF_BRANCH = "master"
```

Comment:
This is not a logical setting name, I'd expect `open-zaak` repo if I see `self`.